pom.xml

```diff
@@ -571,13 +571,14 @@
                       <!-- Import settings: getting rid of redundant import, forbid star notation, defined order of imports. -->
                       <module name="RedundantImport"/>
                       <module name="AvoidStarImport">
-                        <property name="excludes" value="org.junit.Assert,org.assertj.core.api.Assertions,org.mockito.Mockito"/>
```

Comment:
Why are we scratching this? Is it because of Eclipse code formatting limitations or is it because our current IntelliJ config doesn't have it? (IIRC, it has it)

Reply:
Hi @ge0ffrey, yes, it is due to Eclipse code formatting limitations. This works well in IntelliJ, but I din't found how to do it in Eclipse. So removed it from IntelliJ codestyle (previous commit) and from checkstyle maven plugin as well (this commit).